dcos/subcommand.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import print_function
 
+import hashlib
 import json
 import os
+import platform
 import shutil
+import stat
 import subprocess
 import sys
+import zipfile
 from subprocess import PIPE, Popen
 
+import dcos.http
```

Comment:
`http` imports `config` which imports `subcommand`. http://stackoverflow.com/questions/744373/circular-or-cyclic-imports-in-python